CONTRIBUTING.md

```diff
@@ -77,6 +77,29 @@ mvn -f sdk/{projectForlderDir}/pom.xml -Dgpg.skip clean install
 
 //example: mvn -f sdk/keyvault/azure-security-keyvault-keys/pom.xml clean install
 ```
+### Live testing
+
+Live tests assume a live resource has been created and appropriate environment
+variables have been set for the test process. To automate setting up live
+resources we use created a script called `New-TestResources.ps1` that deploys
+resources for a given service.
+
+To see what resources will be deployed for a live service, check the
+`test-resources.json` ARM template files in the service you wish to deploy for
+testing, for example `sdk\keyvault\test-resources.json`.
+
+To deploy live resources for testing use the steps documented in [`Example 1 of New-TestResources.ps1`](eng/common/TestResources/New-TestResources.ps1.md#example-1)
+to set up a service principal and deploy live testing resources.
+
+To run live tests after deploying live resources:
+
+1. Open Developer Command Prompt
+2. Navigate to service directory e.g. _"sdk\keyvault"_
+3. Invoke `dotnet test`
```

Comment:
Is this example assume to be dotnet as language ? Other languages folks .. Would we give references on how to do for other languages ?

Reply:
It was. I fixed it :)